public/static/js/init.py

```diff
@@ -732,13 +758,17 @@ def run_turtle_cmd(msg):
 
 def hit_breakpoint(lineno, alocals, aglobals):
     global step_into
+    global last_seen_lineno
+    if last_seen_lineno == lineno:
+        return True
+    last_seen_lineno = lineno
```

Comment:
if we have just seen (not necessarily) stopped on this line. Could happen on a while line where there is a hit_break before the while and inside the condition